mkdocs/overrides/devbook/transactions/complete-aggregate.py

```diff
@@ -0,0 +1,174 @@
+		median_mult = response_json['medianFeeMultiplier']
+		minimum_mult = response_json['minFeeMultiplier']
+		fee_mult = max(median_mult, minimum_mult)
+		print(f'  Fee multiplier: {fee_mult}')
+
+	# Account A sends 10 XYM to Account B
+	embedded_transaction_1 = facade.transaction_factory.create_embedded({
+		'type': 'transfer_transaction_v1',
+		'signer_public_key': account_a_key_pair.public_key,
+		'recipient_address': account_b_address,
+		'mosaics': [{
+			'mosaic_id': generate_mosaic_alias_id('symbol.xym'),
+			'amount': 10_000_000  # 10 XYM (divisibility = 6)
+		}]
+	})
+
+	# Account B sends 1 custom mosaic to Account A
+	custom_mosaic_id = 0x6D1314BE751B62C2
+	embedded_transaction_2 = facade.transaction_factory.create_embedded({
+		'type': 'transfer_transaction_v1',
+		'signer_public_key': account_b_key_pair.public_key,
+		'recipient_address': account_a_address,
+		'mosaics': [{
+			'mosaic_id': custom_mosaic_id,
+			'amount': 1  # 1 custom mosaic (divisibility = 0)
+		}]
+	})
+
+	# Build the aggregate transaction
+	embedded_transactions = [
+		embedded_transaction_1, embedded_transaction_2]
+	transaction = facade.transaction_factory.create({
+		'type': 'aggregate_complete_transaction_v3',
+		'signer_public_key': account_a_key_pair.public_key,
+		'deadline': timestamp.add_hours(2).timestamp,
+		'transactions_hash': facade.hash_embedded_transactions(
+			embedded_transactions),
+		'transactions': embedded_transactions
+	})
+	# Reserve space for one cosignature (104 bytes each)
+	# and calculate fee for the final transaction size
+	transaction.fee = Amount(fee_mult * (transaction.size + 104))
+	print('Built aggregate transaction:')
+	print(json.dumps(transaction.to_json(), indent=2))
+
+	# --- ACCOUNT A (Initiator) ---
+	print('[Account A] Signing the aggregate...')
+	signature_a = facade.sign_transaction(account_a_key_pair, transaction)
+	transaction_payload = facade.transaction_factory.attach_signature(
+		transaction, signature_a)
+	payload_formatted = json.dumps(
+		json.loads(transaction_payload), indent=2)
+	print(f'[Account A] Payload ready to share:\n{payload_formatted}')
+
+	# --- OFF-CHAIN COORDINATION ---
+	# Account A sends the payload to Account B
+	shared_payload = transaction_payload
+	print('[Sharing] Payload sent to Account B...')
+
+	# --- ACCOUNT B (Cosignatory) ---
+	print('[Account B] Received payload')
+	received_transaction = facade.transaction_factory.deserialize(
+		bytes.fromhex(json.loads(shared_payload)['payload']))
+
+	print('[Account B] Cosigning...')
+	cosignature_b = facade.cosign_transaction(
+		account_b_key_pair, received_transaction)
+	print(f'[Account B] Cosignature created: {cosignature_b}')
+
+	# --- OFF-CHAIN COORDINATION ---
+	# Account B sends the cosignature back to Account A
+	shared_cosignature = cosignature_b
+	print('[Sharing] Cosignature sent back to Account A...')
+
+	# --- ACCOUNT A (Initiator) ---
+	print('[Account A] Received cosignature')
```

Comment:
```suggestion
	print('[Account B] <== Cosignature sent back to Account A (offchain)')

	# --- ACCOUNT A (Initiator) ---
```

Again, use arrows and remove second print.